Lib/multiprocessing/shared_memory.py

```diff
@@ -76,6 +76,8 @@ def __init__(self, name=None, create=False, size=0):
             raise ValueError("'size' must be a positive integer")
         if create:
             self._flags = _O_CREX | os.O_RDWR
+        if create and size == 0:
```

Comment:
I meant make lines 77-80 look like:
```
if create:
    self._flags = _O_CREX | os.O_RDWR
    if size == 0:
        raise ValueError("'size' of new segment must be greater than 0")
```
I'm just trying to group together all of the code involving the test for "create". And I guess the error message should be changed to say "cannot specify a 'size' of zero when creating a new segment", or something like that. The less than zero case has already been handled.

Reply:
Oh ok, did the same.